app/src/main/java/combi/example/andre/finalproject/DownloadUrl.java

```diff
@@ -0,0 +1,53 @@
+package combi.example.andre.finalproject;
+
+import android.util.Log;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+
+public class DownloadUrl {
+
+    public String readUrl(String strUrl) throws IOException{
+        String data = "";
+        InputStream iStream = null;
+        HttpURLConnection urlConnection = null;
+        try{
+            URL url = new URL(strUrl);
+
+            urlConnection = (HttpURLConnection) url.openConnection();
+
+            urlConnection.connect();
+
+            iStream = urlConnection.getInputStream();
+

```

Comment:
It might be easy to read while you are coding, but every newline on each line is a bit too much. Later on please reformat your code

![2018-06-03 17_29_49-myapplication3 c__users_seiry_documents_github_myapplication3 - _app_src_ma](https://user-images.githubusercontent.com/1240672/40885620-0dc6b9a4-6754-11e8-8b6d-94d454aee60e.png)